renovate.json

```diff
@@ -7,9 +7,6 @@
     "constraints": {
       "go": "1.20"
     },
-    "extends": [
-      "config:base"
-    ],
```

Comment:
I cant find a definition for `base` anywhere.  I don't think this is doing anything, but I'd like to be proven wrong.

Reply:
Huh... it was definitely in the config example i found, but i wonder if we should change this to another preset https://docs.renovatebot.com/presets-config/#configrecommended

Reply:
hah, it looks like it `base` -> `recommended` https://github.com/renovatebot/renovate/blob/5ff18174495251d2e3e204be770547776a14fa9c/lib/config/presets/common.ts#L17

Reply:
added it back as `config:recommended`